outputs.tf

```diff
@@ -1,4 +1,91 @@
-# output "variable" {
-#   description = "output variable description"
-#   value       = variable.main.name
-# }
+output "id" {
+  description = "A unique ID that can be used to identify and reference a Kubernetes cluster."
+  sensitive   = false
+  value       = digitalocean_kubernetes_cluster.main.id
+}
+
+output "cluster_subnet" {
+  description = "The range of IP addresses in the overlay network of the Kubernetes cluster."
+  sensitive   = false
+  value       = digitalocean_kubernetes_cluster.main.cluster_subnet
+}
+
+output "service_subnet" {
+  description = "The range of assignable IP addresses for services running in the Kubernetes cluster."
+  sensitive   = false
+  value       = digitalocean_kubernetes_cluster.main.service_subnet
+}
+
+output "ipv4_address" {
+  description = "The public IPv4 address of the Kubernetes master node."
+  sensitive   = false
+  value       = digitalocean_kubernetes_cluster.main.ipv4_address
+}
+
+output "endpoint" {
+  description = "The base URL of the API server on the Kubernetes master node."
+  sensitive   = false
+  value       = digitalocean_kubernetes_cluster.main.endpoint
+}
+
+output "auto_upgrade" {
+  description = "A boolean value indicating whether the cluster will be automatically upgraded to new patch releases during its maintenance window."
+  sensitive   = false
+  value       = digitalocean_kubernetes_cluster.main.auto_upgrade
+}
+
+output "kube_config" {
```

Comment:
Maybe split this to separate outputs for host,endpoint and ca-cert? The other atrributes doesn't seems usable (ie. raw_config).
Also In this form i believe it's sensitive as it holds `token` field and raw config with embedded token.